pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -665,6 +665,7 @@ type AzureCloudSpec struct {
 	SubnetName            string `json:"subnet"`
 	RouteTableName        string `json:"routeTable"`
 	SecurityGroup         string `json:"securityGroup"`
+	AllowedIPRange        string `json:"allowedIPRange,omitempty"`
```

Comment:
Sorry for not raising this during the initial review I did, but I think we should either a) rename this field to include a reference to this being related to node ports (e.g. `NodePortsAllowedIPRange`) or b) add a description to the API object to clarify what's it for (that happens through a comment afaik, no?). Perhaps both? There are many things in a cloud provider spec that "allowed IP ranges" could refer to.

```suggestion
        // AllowedIPRange defines the IP range that is allowed to access node ports via firewall/security groups. Defaults to "0.0.0.0/0" (access allowed from any IP)
	AllowedIPRange        string `json:"allowedIPRange,omitempty"`
```

Obviously applies to the other additions to provider cloud specs too.